.gitignore

```diff
@@ -1,3 +1,15 @@
+#############################################################
+# Keep in might that sub-directories might contain additional
+# .gitignore files to exclude specific files.
+#############################################################
+
+# Ignore test output - this will also make sure that you do not accidentally add large files to your commit
+*output*.nur
```

Comment:
I think we discussed to make it even more agressive and exclude all *.nur and *.hdf5 so that people don't accidentally commit large files. Other than that it looks good.